SPECS/kernel/config_aarch64

```diff
@@ -881,7 +987,8 @@ CONFIG_MMU_NOTIFIER=y
 CONFIG_KSM=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_ARCH_SUPPORTS_MEMORY_FAILURE=y
-# CONFIG_MEMORY_FAILURE is not set
+CONFIG_MEMORY_FAILURE=y
+CONFIG_HWPOISON_INJECT=m
```

Comment:
This config has been identified by the kernel config hardening checker and should be disabled

Reply:
Discussed offline. May be required for boot. Will prune down in future PR.